test/test_datasets.py

```diff
@@ -1920,7 +1917,8 @@ def test_flow(self):
         with self.create_dataset(split="train") as (dataset, _):
             assert dataset._flow_list and len(dataset._flow_list) == len(dataset._image_list)
             for _, _, flow in dataset:
-                assert flow == self._FAKE_FLOW
+                assert flow.shape == (2, self.FLOW_H, self.FLOW_W)
```

Comment:
Instead of passing this information through class variables, `inject_fake_data` can also return a dictionary. If you return `dict(num_examples=..., flow_shape=...)` there, you can access it here with

```python
with self.create_dataset(...) as (dataset, info):
    ...
    assert flow.shape == (2, *info["flow_shape"])
```